platforms/corda-ent/charts/idman/values.yaml

```diff
@@ -1,67 +1,57 @@
-# Default values for Identity Manager service.
+# Default values for Identity Manager (Idman) service.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-deployment:
-  annotations: {}
-
+#############################################################
+#                   Basic Configuration                     #
+#############################################################
 # Provide the name of the node
 # Eg. nodeName: idman
 nodeName:
 
-# This section contains the Enterprise-Corda node metadata.
+# This section contains the Corda Enterprise Idman metadata.
 metadata:
-  # Provide the namespace for the Corda node.
+  # Provide the namespace for the Corda Enterprise Idman.
   # Eg. namespace: cenm
   namespace:
-  # Provide the labels to the Corda node.
+  # Provide any additional labels for the Corda Enterprise Idman.
   labels:
 
-pvc:
-  #   annotations:
-  #     key: "value"
-  annotations: {}
-
-# This section contains the storage information.
-storage:
-  # Provide the name of the storageclass.
-  # NOTE: Make sure that the storageclass exist prior to this deployment as
-  # this chart doesn't create the storageclass.
-  # Eg. name: cenm
-  name:
-
-# Provide the number of replicas for your pods
-# Eg. replicas: 1
-replicas:
-
-# Provide image for init container
+# Provide information regarding the Docker images used.
 image:
-  # Provide the alpine utils image.
+  # Provide the alpine utils image, which is used for all init-containers of deployments/jobs.
   # NOTE: The alpine image used is the base alpine image with CURL installed.
   # Eg. initContainerName: hyperledgerlabs/alpine-utils:1.0
   initContainerName:
+  # Provide the image for the main Idman container.
+  # Eg. idmanContainerName: corda/enterprise-identitymanager:1.2-zulu-openjdk8u242
+  idmanContainerName:
   # Provide the docker-registry secret created and stored in kubernetes cluster as a secret.
   # Eg. imagePullSecret: regcred
-  imagepullsecret:
-
-# Provide image for idman container
-dockerImage:
-  # EX. name: corda/enterprise-identitymanager
-  name: 
-  # Ex tag: 1.2-zulu-openjdk8u242
-  tag: 
-  # Ex. pullPolicy: Always
-  pullPolicy: 
-  # EX. imagePullSecret: cenm-registry
   imagePullSecret: 
+  # Pull policy to be used for the Docker image
+  # Eg. pullPolicy: Always
+  pullPolicy: 
+  
+# This section contains the storage information, used for the Persistent Volume Claims (PVC).
+storage:
+  # Provide the name of the storageclass.
+  # NOTE: Make sure that the storageclass exist prior to this deployment as
+  # this chart doesn't create the storageclass.
+  # Eg. name: cenm
+  name:
```

Comment:
Storage sizes are mentioned here and not hardcoded in pvc.yaml

Reply:
Fixed in [ae9a768](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/992/commits/ae9a76899ddcce9ea9e2067c3ba79ebc784c08c5)

Reply:
yes, for now hardcoding in tpl is fine as we do not want too many configs in network.yaml